src/seqpacket.rs

```diff
@@ -650,6 +662,17 @@ impl NonblockingUnixSeqpacketConn {
         // nonblockingness is shared and therefore inherited
         Ok(NonblockingUnixSeqpacketConn { fd: cloned.into_raw_fd() })
     }
+
+    /// Shuts down the read, write, or both halves of this connection.
+    pub fn shutdown(&self, how: Shutdown) -> io::Result<()> {
```

Comment:
Thanks for adding `shutdown()`!

Maybe add a test? (I can do that later though)